<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1432</width>
    <height>920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>1281</width>
      <height>581</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>144</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="triangleBtn">
       <property name="text">
        <string>треугольник</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="circleBtn">
       <property name="text">
        <string>круг</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="squareBtn">
       <property name="text">
        <string>прямоугольник</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rombusBtn">
       <property name="text">
        <string>ромб</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="hexagonBtn">
       <property name="text">
        <string>шестиугольник</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="star1Btn">
       <property name="text">
        <string>звезда 5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="star2Btn">
       <property name="text">
        <string>звезда 6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lineBtn">
       <property name="text">
        <string>линия</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>680</y>
      <width>791</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Периметр</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QTextBrowser" name="Perimetr"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Центр х</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Центр у</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QTextBrowser" name="centerX"/>
     </item>
     <item row="2" column="3">
      <widget class="QTextBrowser" name="centerY"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Площадь</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTextBrowser" name="Square"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>751</width>
      <height>90</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QPushButton" name="rotateRightBtn">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>↻</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="rotateLeftBtn">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>↺</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="sizePlusBtn">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="2">
      <widget class="QPushButton" name="movingBtn">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>⌜   ⌝
⌞   ⌟</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="sizeMinusBtn">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="clearBtn">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>700</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1432</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
